appinventor/blocklyeditor/src/versioning.js

```diff
@@ -1527,7 +1527,10 @@ Blockly.Versioning.AllUpgradeMaps =
 
     // The LoadingDialogMessage property was added
     // The ShowLoadingDialog property was added
-    4: "noUpgrade"
+    4: "noUpgrade",
```

Comment:
Since we are no longer relying on upgrading to trigger the warning, this should be removed (the control version is still 4).